src/BenchmarksServer/Startup.cs

```diff
@@ -262,7 +266,8 @@ private static void AddSourceDependencies(string path, string benchmarksDir, IEn
             benchmarksProjectDocument.Root.Add(netFrameworkReferences);
             benchmarksProjectDocument.Root.Add(netCoreReferences);
 
-            using (var writer = XmlWriter.Create(benchmarksProjectPath, new XmlWriterSettings { Indent = true, OmitXmlDeclaration = true }))
+            using (var stream = File.OpenWrite(benchmarksProjectPath))
+            using (var writer = XmlWriter.Create(stream, new XmlWriterSettings { Indent = true, OmitXmlDeclaration = true }))
```

Comment:
We don't have the former API now that we've reverted to a previous version of NETStandard.Library.